### DISCUSSION TITLE
how to use the Lokr lora on diffusion pipeline or flux pipeline

### DISCUSSION THREAD

**Discussion by michaelcyshield** [bghira/SimpleTuner#887] [Q&A]
Hello, I found this inference piece of code on hugging face but when I used it, I did not get the same results as the validation outputs, is there something I am missing? additionally to the code in this link, I quantize to fp8 using the optima quanto library, however during training its set so int8, could this be a potential issue? also I realized that I need to quantize after the merge of the lokr file and not before.
does anyone know to to perfectly recreate the inference done during validation while training the lokr?
[https://huggingface.co/terminusresearch/flux-lokr-garfield-nomask](url)

**Comment by bghira** [comment]
chances are you can never perfectly recreate those outputs. it's just how things go

**Reply by michaelcyshield** [reply]
I have quantised using the same quantisation as training, and set the lora scale in both training and inference to 1, it seems that any other lora scale is impossible to reproduce for some reason even if you set them to be the same in train and inference, 1 is the only number that gives me some reliability.
